File: figureGeometric/figure.java
public class figure {
    public static void main (String [] args){

    figureGeometric square = new square(3);
    figureGeometric circle = new circle(5);
    figureGeometric triangle = new triangle(5,8);
    figureGeometric rectangle = new rectangle(4, 5);

    System.out.println("Area of square: "+square.getNameSpanish()+"::"+square.calculateArea());
    System.out.println("Area of circle: "+circle.getNameSpanish()+"::"+circle.calculateArea());
    System.out.println("Area of triangle: "+triangle.getNameSpanish()+"::"+triangle.calculateArea());
    System.out.println("Area of rectangle: "+square.getNameSpanish()+"::"+rectangle.calculateArea());



    }
    
}

File: figureGeometricphp - copia/php/figure.php
<?php

include('FigureGeometria.php');
include('square.php');
include('triangule.php');
include('circle.php');
include('rectangule.php'); 

class Figure {
    public function __construct() {
        
        $square = new Square(5); 
        $triangule = new Triangule(4, 6); 
        $circle = new Circle(7);
        $rectangule = new Rectangule(5, 8); 
        
        echo "Área del Cuadrado: " . $square->getNameSpanish() . " :: " . $square->getCalculateArea() . "<br>";
        echo "Área del Triángulo: " . $triangule->getNameSpanish() . " :: " . $triangule->getCalculateArea() . "<br>";
        echo "Área del Círculo: " . $circle->getNameSpanish() . " :: " . $circle->getCalculateArea() . "<br>";
        echo "Área del Rectángulo: " . $rectangule->getNameSpanish() . " :: " . $rectangule->getCalculateArea() . "<br>";
    }
}


new Figure();
?>

